### PR TITLE
Reorg cap deployment guide as all-in-one

### PR DESCRIPTION
**Pull request by cjschroder**
deployment, admin, and user guide. Reasons for combined guides:
--Customer preference for single product manual
--Easier to search in single HTML
--Multi-chapter HTML provides collapsible/expandable TOC

### CODE REVIEW

File: xml/common_intro_available_doc_i.xml [hunk 1]
```diff
@@ -41,27 +41,13 @@
...
   <varlistentry os="sles;sled">
-   <term><xref linkend="book.cap.deployment"/>
+   <term><xref linkend="book.cap.guides"/>
    </term>
    <listitem>
     <para>
-     The &productname; deployment guide gives you details about installation
-     and configuration of &productname; along with a description of
-     architecture and minimum system requirements.
+     The &productname; guide is a comprehensive guide providing deployment, administration, and user guides, and      architecture and minimum system requirements.
```
Comment (btat) [inline xml/common_intro_available_doc_i.xml]: There are some extra spaces between "and" and "architecture"

**Comment by btat** [comment]
LGTM :+1:

File: xml/common_intro_available_doc_i.xml (hunk 1 above)
Comment (cjschroder) [reply xml/common_intro_available_doc_i.xml]: I will fix that and run daps-xmlformat in a separate commit.

**Comment by cjschroder** [comment]
To add or move chapters, just edit book_cap_guides.xml

**Comment by cjschroder** [comment]
another advantage of a single combined manual-- much easier to support multiple CAP versions, if we ever have to

**Pull request merged**